tests/conftest.py

```diff
@@ -887,6 +887,41 @@ def _fetch_intermittent_bugs(additional_params, limit, duplicate_chain_length):
     )
 
 
+class MockResponse:
+    def __init__(self):
+        self.status_code = 200
+        self.content = {
+            "artifacts": [
+                {
+                    "storageType": "something",
```

Comment:
I think it would be nice to have some actual values here instead of `something`.